lib/TPP/GPU/LinalgToXeGPU.cpp

```diff
@@ -196,7 +196,7 @@ static std::optional<Value> lowerGenericOp(linalg::GenericOp genericOp,
     }
 
     auto zeroVec =
-        rewriter.create<vector::BroadcastOp>(loc, resType, zeroConst);
+        rewriter.create<mlir::vector::BroadcastOp>(loc, resType, zeroConst);
```

Comment:
If this change is necessary now (it wasn't before), then you changed something in the namespace resolution. Make sure you remove the change and these lines can continue as they were.

Always revisit your assumptions when a file that should remain untouched needs to be touched. Often you're doing something unnecessary elsewhere.

Reply:
My bad, that was a stray change, I'll get rid of it.

Reply:
I have reverted the changes to both LinalgToXeGPU and GPUVectorize, theyre no longer needed as I've changed the dialect dependence to mlir::vector::VectorDialect instead of vector::VectorDialect.